app/views/content/non-uk-teachers/get-an-international-relocation-payment.md

```diff
@@ -0,0 +1,318 @@
+---
+title: "Get an international relocation payment (IRP) worth £10,000"
+description: |-
+  Non-UK teachers and trainees in physics and languages could be eligible for financial help from the UK government.
```

Comment:
makes sense to include 'international relocation payment' in the description to help with search, if this is something users hear about and search for in Google?

Reply:
Good idea